BinaryOracle/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,83 @@
+# Contributor Covenant Code of Conduct
+
```

Comment:
Please remove .env.example, .gitignore, CODE_OF_CONDUCT, LICENSE, hardhat.config.js and package.json files